streams/src/test/java/org/apache/kafka/streams/processor/internals/StateDirectoryTest.java

```diff
@@ -88,6 +88,25 @@ public void shouldCreateTaskStateDirectory() {
         assertTrue(taskDirectory.isDirectory());
     }
 
+    @Test
+    public void shouldNotCreateTaskStateDirectory() {
+        final File stateDir = new File(TestUtils.IO_TMP_DIR, TestUtils.randomString(5));
```

Comment:
`TestUtils.tempDirectory` actually creates the directory. To test some case, `baseDir` shouldn't be created.

Reply:
I see. Thanks for pointing out.

Can we add prefix `"kafka-"` before the random String though? If cleanup fails, it would be helpful for users to see that it's related to kafka if they want to clean it up.

Reply:
prefix added.